traits/ctrait.py

```diff
@@ -109,6 +111,37 @@ def comparison_mode(self):
     def comparison_mode(self, value):
         ctraits.cTrait.comparison_mode.__set__(self, value)
 
+    @property
+    def property(self):
+        """ Return a tuple of callables (fget, fset, validate) for the
+        property trait."""
+        return self._get_property()
+
+    @property.setter
+    def property(self, value):
+        """ Set the fget, fset, validate callables for the property.
+
+        Parameters
+        ----------
+        value : tuple
+            Value should be the tuple of callables (fget, fset, validate).
+
+        """
+        func_arg_counts = []
+
+        for arg in value:
+            if arg is None:
+                func_arg_counts.extend([None, 0])
+            else:
+                try:
+                    sig = inspect.signature(arg)
+                except TypeError:
+                    raise TypeError("Property setter expects a callable "
```

Comment:
I'd suggest dropping the `try / except` and letting the `TypeError` from `inspect.signature` propagate directly. The message from `inspect.signature` is fairly self-explanatory (for example: `TypeError: 3.141592653589793 is not a callable object`).

Reply:
ok